packages/flutter_tools/lib/src/ios/setup_xcodeproj.dart

```diff
@@ -104,36 +61,44 @@ bool xcodeProjectRequiresUpdate(BuildMode mode) {
 }
 
 Future<int> setupXcodeProjectHarness(String flutterProjectPath, BuildMode mode, String target) async {
-  // Step 1: Fetch the archive from the cloud
+  // Step 1: Copy templates into user project directory
   String iosFilesPath = path.join(flutterProjectPath, 'ios');
-  String xcodeprojPath = path.join(iosFilesPath, '.generated');
-
-  Directory toolDir = tools.getEngineArtifactsDirectory(TargetPlatform.ios, mode);
-  File archiveFile = new File(path.join(toolDir.path, 'FlutterXcode.zip'));
-  List<int> archiveBytes = archiveFile.readAsBytesSync();
-
-  if (archiveBytes.isEmpty) {
-    printError('Error: No archive bytes received.');
-    return 1;
-  }
-
-  // Step 2: Inflate the archive into the user project directory
-  bool result = _inflateXcodeArchive(xcodeprojPath, archiveBytes);
-  if (!result) {
-    printError('Could not inflate the Xcode project archive.');
-    return 1;
-  }
+  String xcodeProjPath = path.join(iosFilesPath, '.generated');
+  String templatesPath = path.join(Cache.flutterRoot, 'packages', 'flutter_tools', 'templates', 'build-ios');
+  _copyFolderSync(templatesPath, xcodeProjPath);
 
-  // Step 3: Populate the Generated.xcconfig with project specific paths
+  // Step 2: Populate the Generated.xcconfig with project specific paths
   updateXcodeGeneratedProperties(flutterProjectPath, mode, target);
 
-  // Step 4: Write the REVISION file
-  File revisionFile = new File(path.join(xcodeprojPath, 'REVISION'));
+  // Step 3: Write the REVISION file
+  File revisionFile = new File(path.join(xcodeProjPath, 'REVISION'));
   revisionFile.createSync();
   revisionFile.writeAsStringSync('${Cache.engineRevision}-${getModeName(mode)}');
 
-  // Step 5: Tell the user the location of the generated project.
+  // Step 4: Tell the user the location of the generated project.
   printStatus('Xcode project created in $iosFilesPath/.');
 
   return 0;
 }
+
+void _copyFolderSync(String srcPath, String destPath) {
```

Comment:
> I take it https://www.dartdocs.org/documentation/file/0.1.0/file/FileSystemEntity/copy.html doesn't work?

Correct (UnsupportedOperationError).  I thought about implementing this there but decided to avoid shaving that yak.